plugins/reach/lib/api/data/vendorCatalogItem/KalturaVendorCatalogItem.php

```diff
@@ -97,6 +97,31 @@ abstract class KalturaVendorCatalogItem extends KalturaObject implements IRelate
 	 */
 	public $vendorData;
 
+	/**
+	 * @var KalturaVendorCatalogItemStage
+	 */
+	public $stage;
+
+	/**
+	 * @var int
+	 */
+	public $bulkUpdateId;
```

Comment:
this field is not fully clear to me, its the one that originally updated it? the last one?
why its interesting to expose it if there can be multiple updates?